src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -392,7 +392,18 @@
     {% if controller.class is defined -%}
         {%- if method|default(false) %}<span class="sf-toolbar-status sf-toolbar-redirection-method">{{ method }}</span>{% endif -%}
         {%- set link = controller.file|file_link(controller.line) %}
-        {%- if link %}<a href="{{ link }}" title="{{ controller.class }}">{% else %}<span title="{{ controller.class }}">{% endif %}
+
+        {%- if controller.classFile is defined and controller.method %}
+            {%- set classLink = controller.classFile|file_link(controller.classLine) %}
+            <a href="{{ classLink }}" title="{{ controller.class }}">
+                {{- controller.class|abbr_class|striptags -}}
```

Comment:
Why stripping tags here ?

Reply:
It was like this before, so i'm not sure about the intention but thought i keep it because the original author might have had a reason.